modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_4/PollsPortletIdToDDMPortletIdUpgradeProcess.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.internal.upgrade.v5_1_4;
+
+import com.liferay.dynamic.data.mapping.constants.DDMPortletKeys;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.BasePortletIdUpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Rebeca Silva
+ */
+public class PollsPortletIdToDDMPortletIdUpgradeProcess
+	extends BasePortletIdUpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		runSQL(
+			StringBundler.concat(
+				"delete from Portlet where portletId = '",
+				_PORTLET_ID_POLLS_DISPLAY, "' OR portletId = '",
+				_PORTLET_ID_POLLS, "'"));
+		runSQL(
+			StringBundler.concat(
+				"delete from ResourcePermission where name = '",
+				_PORTLET_ID_POLLS_DISPLAY, "' OR name = '", _PORTLET_ID_POLLS,
+				"'"));
```

Comment:
@rebsilva could you please put the `_PORTLET_ID_POLLS_DISPLAY` after the `_PORTLET_ID_POLLS` to keep the alphabetical order?

Reply:
Sure!